EmpleoDotNet/Services/Social/Twitter/TwitterService.cs

```diff
@@ -41,7 +41,7 @@ public TwitterService()
 
             var message = $"Se busca: {title} http://emplea.do/JobOpportunity/Detail/{UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)}";
 
-            await PostTweet(message);
+            await PostTweet(message).ConfigureAwait(false);
```

Comment:
Efrain beat me to it. Básicamente: 
- Si comienzas un método async, completa la llamada completa en el call stack async (para que no bloquees el thread)
- `ConfigureAwait(false)` en realidad es `.ConfigureAwait(continueOnCapturedContext: false);` para que no se capture el contexto actual, sino que cualquier otro thread pueda recoger la llamada una vez esté completada

More info here as well: 
https://msdn.microsoft.com/en-us/magazine/jj991977.aspx